docs/customer/cdr.md

```diff
@@ -33,10 +32,18 @@ When viewing CDRs for a specific customer, use the **`Recalc CDR`** button to re
 
 * **Release Reason** (for Rerate operations)- Select the reason for the call's termination (multiple selections allowed). This will revise the amount charged for the calls.
 
-* **Min Duration** (for Adjust Call Duration operations)
+* **Min Duration** (for Adjust Call Duration operations) (Calls with Minimum Duration in the CDR for example 3600 seconds will be considered for re-calculation)
```

Comment:
Minimum Duration of calls that will be considered for re-calculation

Reply:
done